.github/workflows/fossa.yaml

```diff
@@ -0,0 +1,36 @@
+name: fossa
+on:
+  repository_dispatch:
+    types: [fossa]
+  push:
+    branches:
+      - master
+      - release-*
+      - feature/*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+      - release-*
+      - feature/*
+  workflow_dispatch: {}
+jobs:
+  fossa-scan:
```

Comment:
We should add the condition
`if: github.repository_owner == 'dapr' # FOSSA is not intended to run on forks.`

Otherwise it will start running on forks and upload multiple copies of the repo to the FOSSA portal.